audit_log/tests/audit_log_tests/test_logging.py

```diff
@@ -235,3 +215,44 @@ def test_logging(self):
         self.assertEqual(prop.audit_log.all()[1].action_type, 'I')
         self.assertEqual(prop.audit_log.all()[0].owned_by, owner2)
         self.assertEqual(prop.audit_log.all()[1].owned_by, owner1)
+
+class TestProxyModel(TestCase):
+    urls = __name__
+    
+    def test_audit_fields(self):
+        _setup_admin()
+        # Create Normal Item with user 1
+        client_1 = Client()
+        client_1.login(username = "admin@example.com", password = "admin")
```

Comment:
Can we please be consistent - always use single quotes (`'`), not double quotes?